.github/workflows/helpdot-review.yml

```diff
@@ -33,4 +33,5 @@ jobs:
         uses: anthropics/claude-code-action@v1
```

Comment:
We should use the hash of the latest v1 in this follow up too

Reply:
done